src/structures/Message.js

```diff
@@ -418,6 +418,24 @@ class Message extends Base {
     );
   }
 
+  /**
+   * Whether the message is crosspostable by the client user
+   * @type {boolean}
+   * @readonly
+   */
+  get crosspostable() {
+    return (
+      this.channel.type === 'news' &&
+      !this.flags.any([MessageFlags.FLAGS.CROSSPOSTED, MessageFlags.FLAGS.IS_CROSSPOST]) &&
```

Comment:
crossposts can be crossposted
```suggestion
      !this.flags.has(MessageFlags.FLAGS.CROSSPOSTED) &&
```